pkg/migrate/cstor/volume.go

```diff
@@ -534,9 +534,16 @@ func (v *VolumeMigrator) updateStorageClass(pvName, scName string) error {
 			return err
 		}
 	}
+	cond0, ret0 := isTmpSCValid(v, scName)
```

Comment:
Second parameter should be `tmp-migrate- + scName`
```suggestion
	cond0, ret0 := isTmpSCValid(v, "tmp-migrate-" + scName)
```